Examples/test-suite/python/Makefile.in

```diff
@@ -98,6 +97,7 @@ endif
 # Test requiring python-3.11 minimum if using Py_LIMITED_API
 ifeq (1,$(PY_ABI_311))
 CPP_TEST_CASES += pyabi311_bufferinterface
+CPP_TEST_CASES += python_pybuffer
```

Comment:
The python_pybuffer test works before your patch is applied as follows:
```
$ cd Examples/test-suite/python
$ make python_pybuffer.cpptest PY_ABI_VER=3.9 CXXFLAGS=-O1
```
so the patch seems to be a regression - I don't expect to see any changes in this Makefile.in file which no longer runs this test if pyabi < 311.

Reply:
no, this fails with python >=3.13 / abi<3.11 (the case we"re interested in)
```
python_pybuffer_wrap.cxx:3630:2: error: #error "The old Buffer Protocol was removed in python-3.13 and is only part of the stable ABI from python-3.11"
 3630 | #error "The old Buffer Protocol was removed in python-3.13 and is only part of the stable ABI from python-3.11"
      |  ^~~~~
```